src/embeds/NewsEmbed.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
```

Comment:
NewsEmbed, PrimeAccessEmbed, and UpdateEmbed should be combined and add a switch. I don't think they each need a different class when they only vary by title and field name.